src/swagger-generation/output/microsoftgraph-beta-0.1.10-preview.json

```diff
@@ -692,6 +696,10 @@
             "verifiedPublisher": {
               "$ref": "#/definitions/microsoft.graph.verifiedPublisher",
               "description": "Specifies the verified publisher of the application that's linked to this service principal."
+            },
+            "owners": {
+              "$ref": "#/definitions/microsoft.graph.relationship",
+              "description": "Directory objects that are owners of this servicePrincipal. The owners are a set of nonadmin users or servicePrincipals who are allowed to modify this object. Read-only. Nullable"
```

Comment:
```suggestion
              "description": "Directory objects that are owners of this servicePrincipal. The owners are a set of nonadmin users or servicePrincipals who are allowed to modify this object. Read-only. Nullable."
```

Reply:
There are a bunch of places in the output files where the period is missing after nullable.  I imagine fixing once at the source would address them all.